lib/views/index.erb

```diff
@@ -1,30 +1,56 @@
 <% if @my_cart.count != 0 %>
-  <div id="shopping_cart">
-    Shopping cart
+<div class="cell">
```

Comment:
Sorry, I see it now. (was on the wrrong branch - looked in `master` rather than `develop` No drama but please get rid of the commented code in the models.